kitsune-fe/package.json

```diff
@@ -29,7 +33,11 @@
     "lodash": "^4.17.21",
     "pinia": "^2.1.6",
     "pinia-plugin-persistedstate": "^3.2.0",
+    "tiptap-markdown": "^0.8.2",
+    "unhead": "^1.7.4",
     "vue": "^3.2.41",
+    "vue-i18n": "^9.5.0",
+    "vue-powerglitch": "^1.0.0",
```

Comment:
`powerglitch` doesn't seem to respect the [`prefers-reduced-motion`] media query. I doubt if the animation counts as a [motion animation] as mentioned by WCAG 2.1 Success Criterion 2.3.3 (Animation from Interactions), but I think it's worth noting anyway.

[`prefers-reduced-motion`]: <https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion>
[motion animation]: <https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html#dfn-motion-animation>

Reply:
That's a good point. I think I'm gonna wrap this element inside a wrapper that implements a check for that media query

Reply:
Added the wrapper component, pretty simple one and it even reacts live on operating system preference changes